core/src/main/java/com/linecorp/armeria/internal/metric/RequestMetricSupport.java

```diff
@@ -130,13 +134,17 @@ private RequestMetricSupport() {}
     private interface RequestMetrics {
         AtomicInteger active();
 
-        Counter success();
-
         Counter failure();
```

Comment:
we will have separate request_failure and response_failure, which may be a bit hard to understand from user view, but from system view, it's apparently different thing. HDYT about this?